sdk/communication/azure-communication-chat/src/main/java/com/azure/communication/chat/implementation/converters/AttachmentTypeConverter.java

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.communication.chat.implementation.converters;
+
+import com.azure.communication.chat.models.AttachmentType;
+
+public final class AttachmentTypeConverter {
+    public static AttachmentType convert(com.azure.communication.chat.implementation.models.AttachmentType obj) {
+        if (obj == null) {
+            return null;
```

Comment:
we usually fail fast and throw when given invalid input (also would be nice to name parameter with meaningful name like `attachmentType`:

```java
Objects.requireNonNull(attachmentType, "'attachmentType' cannot be null.");
```